ui/config-overrides.js

```diff
@@ -34,20 +34,6 @@ function isBuildAsDevServer() {
   return process.env.NODE_ENV !== 'production'
 }
 
-const enableEslintIgnore = () => (config) => {
-  const eslintRule = config.module.rules.filter(
-    (r) =>
-      r.use && r.use.some((u) => u.options && u.options.useEslintrc !== void 0)
-  )[0]
-  const options = eslintRule.use[0].options
-  options.ignore = true
-  options.ignorePattern = 'lib/client/api/*.ts'
```

Comment:
Why can we remote this ignore pattern?

Reply:
There are lint disable hints in the file. For some unknown reasons previously they are not taking effect:

```typescript
/* tslint:disable */
/* eslint-disable */
```

Reply:
Oops..